src/index.scss

```diff
@@ -7,3 +7,52 @@ html {
 *::after {
   box-sizing: inherit;
 }
+
+.wrapper {
+  padding: 0.5rem;
+}
+
+.email-client {
+  border: 1px solid #ccc;
+  display: flex;
+  height: 100vh;
```

Comment:
`height: calc(100vh - 2rem)` unless you’re leaving that as an exercise to the reader

Reply:
Hehehe it was a struggle to know when to stop. But maybe I go ahead and throw that in since this is js group not css group.